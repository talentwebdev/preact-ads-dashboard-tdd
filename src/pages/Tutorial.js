import React from 'react'

function Tutorial() {
    return (
        <div>
            This is a tutorial page
        </div>
    )
}

export default Tutorial
