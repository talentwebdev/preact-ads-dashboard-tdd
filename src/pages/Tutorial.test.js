import {shallow} from 'enzyme';
import Tutorial from './Tutorial';

describe('Tutorial', () => {
    let tutorialContainer;

    beforeEach(() => tutorialContainer = shallow(<Tutorial />));

    test('should render the div element', () => {
        expect(tutorialContainer.find('div').length).toBeGreaterThanOrEqual(1);
    });
});