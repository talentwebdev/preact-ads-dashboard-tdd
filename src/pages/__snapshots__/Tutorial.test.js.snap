// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tutorial renders Tutorial correctly 1`] = `ShallowWrapper {}`;
